utils/swift_build_support/swift_build_support/products/tensorflow.py

```diff
@@ -136,6 +143,36 @@ def _symlink(dest, src):
     os.symlink(dest, src)
 
 
+# Remove the first components of a path.
+def _remove_first_path_components(root, prefix_len):
+    return os.path.sep.join(root.split(os.path.sep)[prefix_len:])
+
+
+# Translate given path to destination.
+def _translate_path(root, dest, prefix_len):
+    return os.path.join(dest, _remove_first_path_components(root, prefix_len))
+
+
+def _copy_x10_headers(source, dest, name_pattern):
+    try:
+        os.makedirs(dest)
```

Comment:
You should use the `_silenced` decorator to ensure that subsequent runs are successful.

Reply:
Done.